RELEASE_NOTES.md

```diff
@@ -10,7 +10,18 @@
 
 ## New Features
 
-<!-- Here goes the main new features and examples or instructions on how to use them -->
+* The `MovingWindow` as well as the `OrderedRingBuffer` are having new public
+  methods that are returning the oldest and newest timestamp of all stored samples.
+
+* The `OrderedRingBuffer`s `wrap` and `normalize_timestamp` are now public.
+
+* The `PeriodicFeatureExtractor` has been added.
+
+  This is a tool to create certain profiles out of periodic reoccurring windows
+  inside a `MovingWindow`.
+
+  As an example one can create a daily profile of specific weekdays which will
+  be returned as numpy arrays.
```

Comment:
Reformatted because of how notes are rendered in releases (GitHub keep the end of lines) and removed references to `OrderedRingBuffer` as it is not public (so users don't care about changes in it):

```suggestion
* The `MovingWindow` has new public methods that are returning the oldest and newest timestamp of all stored samples.

* The `PeriodicFeatureExtractor` has been added.

  This is a tool to create certain profiles out of periodic reoccurring windows inside a `MovingWindow`.

  As an example one can create a daily profile of specific weekdays which will be returned as numpy arrays.
```